README.md

```diff
@@ -1,3 +1,5 @@
 # glibc-bootstrap
```

Comment:
Can this get a relatively similar format to e.g. https://github.com/homebrew/homebrew-bundle in terms of sections?

Reply:
Definitely.  I wasn’t really sure which existing repo was the best template to use  but homebrew-bundle seems like a good starting point.

Reply:
I did some reformatting to resemble the example you shared.  One thing we probably need to add is a license.  Should we use MIT (what homebrew-bundle uses) or BSD 2-clause (what homebrew-core and brew use)?

Reply:
BSD-2 clause I reckon.